integration/simulation/network/geth_network.go

```diff
@@ -142,14 +142,14 @@ func (n *networkInMemGeth) Create(params *params.SimParams, stats *stats.Stats)
 }
 
 func (n *networkInMemGeth) TearDown() {
-	defer n.gethNetwork.StopNodes()
```

Comment:
Ok cool, so the removing of the defers is because they seem unnecessary/confusing rather than to fix the issues, that's fair. I put them in because we were paranoid about panics causing things not to execute but seems unlikely in this method.